src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -112,13 +112,18 @@ def get_fuzz_task_payload(platform=None):
     queue_override = environment.get_value('QUEUE_OVERRIDE')
     platform = queue_override if queue_override else environment.platform()
 
+  platforms = [platform]
+  base_platform = platform.split(':')[0]
+  # Generalized queue for platforms with a base platform (e.g. ANDROID)
+  if base_platform != platform:
+    platforms.append(base_platform)
   if environment.is_local_development():
     query = data_types.FuzzerJob.query()
     query = query.filter(data_types.FuzzerJobs.platform == platform)
```

Comment:
nit: change this to IN too. 

OR actually, can we restructure the code so that there the query setup is only done once between the two branches?